Documentation/ceph-block.md

```diff
@@ -142,6 +142,150 @@ kubectl delete -n rook-ceph cephblockpools.ceph.rook.io replicapool
 kubectl delete storageclass rook-ceph-block
 ```
 
+## RBD Snapshots
+
+### Snapshot Prerequisites
```

Comment:
Is all this content the same, it just moved files?

Reply:
yes just moved it to ceph-block as its related to RBD